sdk/examples/client/getting_started.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! This examples shows how to get the node info.
+//!
+//! ```sh
+//! cargo run --release --example getting_started
```

Comment:
```suggestion
//! cargo run --release --example client_getting_started
```
For consistency with wallet?